cmd/feedback/feedback.go

```diff
@@ -249,15 +245,12 @@ func runFeedbackCommand(ctx context.Context, clients *shared.ClientFactory, cmd
 	surveyNames, surveyPromptOptions := initSurveyOpts(ctx, clients, SurveyStore)
 
 	if _, ok := SurveyStore[surveyNameFlag]; !ok && surveyNameFlag != "" {
-		return slackerror.New("invalid_survey_name").
-			WithMessage("Invalid feedback name provided: %s", surveyNameFlag).
-			WithRemediation("View the feedback options with %s", style.Commandf("feedback --help", false))
+		return slackerror.New(slackerror.ErrFeedbackNameInvalid).
+			WithMessage("Invalid feedback name provided: %s", surveyNameFlag)
```

Comment:
note: Added `slackerror.ErrFeedbackNameInvalid`. The error includes a default message, but this `WithMessage` adds richer detail with the `surveyNameFlag`.